src/test/java/network/brightspots/rcv/TabulatorTests.java

```diff
@@ -87,6 +92,17 @@ private static boolean fileCompare(String path1, String path2) {
     } catch (IOException e) {
       Logger.log(Level.SEVERE, "Error reading file!\n%s", e.toString());
       result = false;
+    } finally {
+      try {
```

Comment:
Were you seeing some issues with file handles leaking?

Reply:
Yeah, it was making it impossible to delete the Dominion test files. Strange that it wasn't an issue for the normal test files.